pkg/ddevapp/apptypes_test.go

```diff
@@ -23,7 +24,7 @@ func TestApptypeDetection(t *testing.T) {
 		"drupal6":   "misc/ahah.js",
 		"drupal7":   "misc/ajax.js",
 		"drupal8":   "core/scripts/drupal.sh",
-		"wordpress": "wp-login.php",
+		"wordpress": "wp-settings.php",
```

Comment:
I actually thought that @alkymst had said either of these would be legit.

Reply:
@rfay wp-settings is what WP core looks for, so I felt it made sense to align with that.